san-francisco-tourism-solution/build.gradle

```diff
@@ -0,0 +1,33 @@
+apply plugin: 'java'
+
+group = 'com.oracle.javaone.2017.hol.junit5'
+version = '0.0.1-SNAPSHOT'
+
+description = "junit5 with gradle as build system"
+
+sourceCompatibility = 1.8
+targetCompatibility = 1.8
+
+ext.junitVersion = '5.0.0'
+ext.mockitoVersion = '2.9.0'
+ext.mockitoExtensionVersion = '1.0.0-RC2'
+
+
+repositories {
+
+    mavenCentral()
+
+}
+
+dependencies {
+    compile group: 'org.mockito', name: 'mockito-core', version: "$mockitoVersion"
+    compile group: 'name.falgout.jeffrey.testing.junit5', name: 'mockito-extension', version: "$mockitoExtensionVersion"
+    testCompile group: 'org.hamcrest', name: 'hamcrest-all', version: '1.3'
```

Comment:
can you extract the **hamcrestVersion** to a property like you did in the other build.gradle file?

Reply:
by the way this is not required. The only time when you need to extract library version into a variable is when you have multiple dependencies from the same family:
`
testCompile group: 'org.junit.jupiter', name: 'junit-jupiter-api', version: "$junitVersion"    testCompile group: 'org.junit.jupiter', name: 'junit-jupiter-params', version: "$junitVersion"    testCompile group: 'org.junit.jupiter', name: 'junit-jupiter-engine', version: "$junitVersion"
`
This is needed to make sure, that all libs from same family `org.junit.jupiter'` are on the same version.